core/blockchain_sethead_test.go

```diff
@@ -2045,14 +2045,10 @@ func testSetHeadWithScheme(t *testing.T, tt *rewindTest, snapshots bool, scheme
 
 	// Force run a freeze cycle
 	type freezer interface {
-		Freeze() error
```

Comment:
Mark: it reverts part of the commit: https://github.com/ethereum/go-ethereum/pull/28683/files#diff-08bfad029bc0c2a3735434672787d567c1f1fb4c21135fc40011fa313e36ac43